File: JavaScript/8js 3/swapValues.js
/*
|--------------------------------------------------------------------------
| Swap Values
|--------------------------------------------------------------------------
|
*/

function swapValues(args) {
   return ([args[0], args[1]] = [args[1], args[0]]);
}


function swapValues(args) {
   var temp = args[0];
   args[0] = args[1];
   args[1] = temp;
}


function swapValues(args) {
   return args.reverse();
}


console.log(swapValues([1, 2]));
console.log(swapValues([1, 2]));

File: Python/8py 3/swap_values.py
"""
|--------------------------------------------------------------------------
| Swap Values
|--------------------------------------------------------------------------
|
"""

def swap_values(args): 
   temp = args[0]
   args[0] = args[1]
   args[1] = temp


def swap_values(args): 
   args[0], args[1] = args[1], args[0]


def swap_values(args): 
   args.reverse()


print(swap_values([1, 2]))
print(swap_values([1, 2]))